scalameter-core/src/main/scala/org/scalameter/InvocationCount.scala

```diff
@@ -3,29 +3,24 @@ package org.scalameter
 import java.io.File
 import org.scalameter.Key._
 import org.scalameter.execution.invocation.InvocationCountMatcher
-import org.scalameter.execution.invocation.instrumentation.MethodInvocationCounter
-import org.scalameter.execution.invocation.instrumentation.MethodSignature
-import org.scalameter.execution.invocation.instrumentation.Instrumentation
-import org.scalameter.utils.ClassPath
-import scala.collection.Seq
-import scala.collection.mutable
+import org.scalameter.execution.invocation.instrumentation.{Instrumentation, MethodInvocationCounter, MethodSignature}
+import scala.collection.{Seq, mutable}
 
 
 /** Mixin for all [[org.scalameter.Measurer]] implementations that perform any kind of
  *  method invocation counting.
  */
-trait InvocationCount extends Measurer {
+trait InvocationCount extends Measurer[Double] {
```

Comment:
I would not yet worry about such a reporter - that would be an extra feature. As long as the clients can easily create a `Double` invocation count measurer whose outputs existing reporters can handle, we should be fine.

Then, once we're done with this PR, we can optionally add a new html reporter that shows the breakdown in an html table.